build/common/openwisp/settings.py

```diff
@@ -82,6 +82,7 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'pipeline.middleware.MinifyHTMLMiddleware'
```

Comment:
Hi,

I think this is only applicable of the static files, right?
In that case, it's not applicable for the API, only the dashboard.

Can we move it to `https://github.com/openwisp/docker-openwisp/blob/master/build/openwisp_dashboard/module_settings.py` ?

Reply:
Yes it is only applicable for static files, I think we can move this to `build/openwisp_dashboard/module_settings.py` 👍